includes/Story_Post_Type.php

```diff
@@ -727,4 +732,59 @@ public function change_default_title( $data ) {
 		}
 		return $data;
 	}
+
+	/**
+	 * Registers the demo admin menu page.
+	 *
+	 * @return void
+	 */
+	public function add_menu_page() {
+		add_submenu_page(
+			'edit.php?post_type=' . self::POST_TYPE_SLUG,
+			__( 'Demo', 'web-stories' ),
+			__( 'Demo', 'web-stories' ),
+			'edit_posts',
+			'web-stories-demo',
+			static function() {
+				// This function is not actually used.
+				// \Google\Web_Stories\Story_Post_Type::redirect_demo() will redirect to the story editor anyway.
+				return null;
+			},
```

Comment:
```suggestion
			'__return_null', // This function is not actually used. \Google\Web_Stories\Story_Post_Type::redirect_demo() will redirect to the story editor anyway.
```

Reply:
Note that this PR is still a draft and WIP. This particular code here will likely be removed entirely.